asciidoc/client-applications.adoc

```diff
@@ -210,6 +270,416 @@ For details on how to configure multi-data center routing policies for a Causal
 ====
 
 
+[[driver-configuration-examples]]
+=== Examples
+
+The table below provides example code snippets for different deployment configurations.
+Each snippet expects an `auth` variable to have been previously defined, containing the authentication details for that connection.
+
+[.tabbed-example]
+.Connecting to a service
+====
+[.include-with-dotnet]
+======
+The table below illustrates examples of how to connect to a service with routing:
+
+[options="header", cols="a,a,a,a"]
+|===
+|Product
+|Security
+|Code snippet
+|Comments
+
+|Neo4j Aura
+|Secured with full certificate
+|
+[source, csharp]
+----
+GraphDatabase.Driver("neo4j+s://graph.example.com:7687", auth)
+----
+
+If you do not have at least the .NET Driver 4.0.1 patch installed, you will need this snippet instead:
+
+[source, csharp]
+----
+String uri = "neo4j://graph.example.com:7687";
+IDriver driver = GraphDatabase.Driver(uri, auth,
+           o => o.WithEncryptionLevel(EncryptionLevel.Encrypted));
+                 .WithTrustManager(TrustManager.CreateChainTrust()));
```

Comment:
an error I made in the original. This line can be deleted.

Reply:
👍